client/app/components/onboardingGuide/styles.scss

```diff
@@ -0,0 +1,131 @@
+.title {
+  font-size: 1.375rem;
+  font-weight: 600;
+  color: #171717;
+  line-height: 1.875rem;
+
```

Comment:
Lots of values that should potentially be vars. Don't we have means to use vars in the styles atm?